### ISSUE TITLE
Where can I enable registration?

### ISSUE THREAD

**Issue by Shablone** [gramps-project/gramps-web#384]
I got a fresh docker-compose setup and I want to allow registration. Currently a click on a registration submit tells me, that is disabled.

In the first sentence of the [documentation](https://www.grampsweb.org/user-guide/registration/) it just says
`it will need to be enabled manually by a site owner before it can be used to log in`

But how do I do that exactly? I can't find any configuration possiblity for that

Gramps 5.1.6
Gramps Web API 1.5.1
Gramps Web Frontend 24.1.2
locale: en
multi-tree: false
task queue: true

**Comment by Shablone** [comment]
I just understood that the documenation senetence I cited talks about the a registration of a user and not the form itself.
Well in the end I still don't know how I can enable the registration functionality

**Comment by DavidMStraub** [comment]
> Currently a click on a registration submit tells me, that is disabled

That what is disable? Please always quote messages literally.

**Comment by Shablone** [comment]
Sorry, I meant to say: that it is disabled

`Fehler: Registration is disabled`

**Comment by DavidMStraub** [comment]
Are you using a multi-tree setup with `TREE="*"`?

**Comment by Shablone** [comment]
No. I use a default configuration

**Comment by DavidMStraub** [comment]
Have you created an owner account yet? Registration is only possible *after* creating the first account.

**Comment by Shablone** [comment]
Yes. I did the initial setup via the webui. My profile states that I'm administrator

**Comment by Shablone** [comment]
I found the issue. The generate intial user is of the role administrator, not owner.
With the role of owner indead it works

**Comment by Shablone** [comment]
Should I close this current issue and create a new one?
The inital setup should create the first user with the role owner, not admistrator.

**Comment by DavidMStraub** [comment]
I'll move it to the backend.
